kalite/settings/base.py

```diff
@@ -13,8 +15,15 @@
     # Not sure if vars from local_settings are accessed ATM. This is one of
     # the big disadvantage of this whole implicit importing chaos... will be
     # fixed later.
-    from kalite.local_settings import *  # @UnusedWildImport
     from kalite import local_settings
+    # This is not a DeprecationWarning by purpose, because those are
+    # ignored during settings module load time
+    warnings.warn(
+        "Using local_settings is deprecated, please write your own settings "
+        "module, import kalite.settings.base, and point your "
+        "DJANGO_SETTINGS_MODULE to your customized settings or run kalite "
+        "COMMAND --settings=my.settings",
```

Comment:
That's circular :) If local_settings is supposed to deprecated and still usable in the "old way", we have to tell people to put their customized settings in a different module in the future... but making another suggestion based on that.